doc/en/user/source/data/raster/imagemosaic/configuration.rst

```diff
@@ -254,6 +254,9 @@ In addition to the required envelope and location attributes, the schema for the
    * - Wildcard
      - N
      - Wildcard used to specify which files should be scanned by the indexer. (For instance: ".")
+   * - WrapStore
+     - N
+     - By default, Postgresql identifiers can't be longer than 63 chars. Longer names will be truncated to that fixed length. When dealing with multidimensional datasets (for instance: NetCDFs, GRIBs) each variable (NetCDF) or parameter (GRIB) is indexed into a table with the same name. Therefore an atmosphere-absorption-optical-thickness-due-to-particulate-organic-matter-ambient-aerosol-particles NetCDF CF variable will be associated to a table with the same name. Postgresql will truncate that to atmosphere-absorption-optical-thickness-due-to-particulate-orga breaking the one-to-one mapping and therefore breaking the proper functioning. Setting the WrapStore flag to ``true`` will establish a hidden mapping between full long names and truncated table names to support proper working.
```

Comment:
Thx. Fixing the repetition and the name.

Not sure about extending the "topic" to general scenario since Oracle already has its own WrappingStore due to its restrictions which is always being used while this flag is currently used by PostgreSQL only.
https://github.com/geotools/geotools/blob/19.x/modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/catalog/GTDataStoreGranuleCatalog.java#L82

Reply:
This section of documentation is generic, not specific to postgresql, as it is written it seems WrapStore is a Postgresql specific configuration. Is it?

Reply:
At the moment, I would say yes since Oracle already had its own wrapping.

Reply:
Ok then :-)